app/decorators/user_decorator.rb

```diff
@@ -1,9 +1,10 @@
 # frozen_string_literal: true
 
 class UserDecorator < SimpleDelegator
-  AVATAR_URL_FORMAT = 'https://api.adorable.io/avatar/%{email}'
+  AVATAR_URL_FORMAT = 'https://secure.gravatar.com/avatar/%{gravatar_id}'
 
   def avatar_url
-    format(AVATAR_URL_FORMAT, email: email)
+    gravatar_id = Digest::MD5::hexdigest(email)
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/nimblehq/nimble-survey-web/blob/92f393866e6a5b4d7e9b5758169434186451305b/app/decorators/user_decorator.rb#L7"></span>Do not use <code>::</code> for method calls.</td>
    </tr>
  </tbody>
</table>